website/project/__init__.py

```diff
@@ -42,7 +42,7 @@ def new_node(category, title, user, description=None, project=None):
     :return Node: Created node
 
     """
-    category = category.strip().lower()
+    category = '_'.join(category.strip().lower().split(' '))
```

Comment:
I assume this is handling "human-readable" inputs , e.g. "Methods and Measures"? If so, I'm not sure that this is a good idea; what if we change our internal represenation? We should use the internal representation ('methods-measures') for everything except for display.